docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.md

```diff
@@ -74,6 +75,37 @@ prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
 image = pipe(prompt=prompt).images[0]
 ```
 
+You can additionally pass "negative conditions" to steer the generation process like so:
```

Comment:
This feels a bit vague, and I think it could be more explicit in explaining what passing `negative_original_size`, `negative_target_size` and `negative_crops_coords_top_left` do. I think it'd be more impactful if it is immediately clear when the user reads it (ie how do these negative conditions affect generation?).

Reply:
It's similar to how classifier guidance is done. With these negative conditions, the model tends to move away from these and focus on non negative conditions instead. Feel free to suggest a text that you think is best suitable here.

Reply:
Maybe something like:

"You can additionally pass *negative conditions* about an image's size and position to avoid undesirable cropping behavior in the generated image, and improve image resolution. "

Reply:
Perfect! Will use it and resolve the conflicts before merging. Thanks a lot!